DeepSpeech.ipynb

```diff
@@ -1150,73 +1156,88 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's introduce a routine that prints a WER report under a given caption. It'll print the given ```mean``` WER plus summaries of the top ten lowest loss items of the given array of WER result tuples (only items with WER!=0 and ordered by their WER)."
+    "First we need a helper method to return a tuple needed for WER reporting. The returned tuple contains three elements. The first is a list of labels, each item in the list corresponds to a tower's labels for a batch. The second is a list of decodings, each item in the list corresponds to a tower's decodings for a batch. The third is a list of distances, each item in the list corresponds to a tower's distances for a batch. The final is a list of losses, each item in the list corresponds to a tower's losses for a batch."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
    "outputs": [],
    "source": [
-    "def print_wer_report(caption, mean, items=[]):\n",
-    "    print\n",
-    "    print \"#\" * 80\n",
-    "    print \"%s WER: %f\" % (caption, mean)\n",
-    "    if len(items) > 0:\n",
-    "        # Filter out all items with WER=0\n",
-    "        items = [a for a in items if a[2] > 0]\n",
-    "        # Order the remaining items by their loss (lowest loss on top)\n",
-    "        items.sort(key=lambda a: a[3])\n",
-    "        # Take only the first 10 items\n",
-    "        items = items[:10]\n",
-    "        # Order this top ten items by their WER (lowest WER on top)\n",
-    "        items.sort(key=lambda a: a[2])\n",
-    "        for a in items:\n",
-    "            print \"-\" * 80\n",
-    "            print \" - WER:    %f\" % a[2]\n",
-    "            print \" - loss:   %f\" % a[3]\n",
-    "            print \" - source: \\\"%s\\\"\" % a[0]\n",
-    "            print \" - result: \\\"%s\\\"\" % a[1] \n",
-    "    print \"#\" * 80\n",
-    "    print"
+    "def get_results_params(data_set):\n",
```

Comment:
I don't think this function is used any more. I guess this was a leftover from Chris's patch.